src/tyr/route_serializer_osrm.cc

```diff
@@ -1832,29 +1871,28 @@ json::ArrayPtr serialize_legs(const google::protobuf::RepeatedPtrField<valhalla:
 
       // Add banner instructions if the user requested them
       if (options.banner_instructions()) {
-        if (prev_step) {
+        if (prev_step && prev_maneuver) {
```

Comment:
Just ensuring here that we have the prev_maneuver. Actually it should always be present